src/ViewItem.js

```diff
@@ -728,7 +725,44 @@ class ViewItem extends React.Component {
       acc05: areAllFieldsEmpty(Object.values(itemNotes)),
       acc06: areAllFieldsEmpty(Object.values(loanAndAvailability)),
       acc07: areAllFieldsEmpty([...Object.values(holdingLocation), ...Object.values(itemLocation)]),
-      acc08: areAllFieldsEmpty([...get(item, 'electronicAccess', [])]),
+      acc08: areAllFieldsEmpty(Object.values(electronicAccess)),
+    };
+
+    const statisticalCodeContent = (!isEmpty(administrativeData.statisticalCodeIds)
+      ? administrativeData.statisticalCodeIds.map(id => ({ codeId: id }))
+      : [{ codeId: '-' }]);
+
+    const electronicAccessContent = (!isEmpty(electronicAccess.electronicAccess)
+      ? electronicAccess.electronicAccess
+      : [{
```

Comment:
You can combine, string 718, method get can have default value, add as default array this one

Reply:
If I add returned object from electronicAccessContent as a default value to electronicAccess field, it will not pass the "checkIfElementIsEmpty()" check (which check if array is empty) and then "Electronic access" accordion will be opened though it contains an empty table.

Reply:
You can omit this comment, actually, if _.get receive form backend an empty array, so defaultValue would be missed